app/components/LiveDock.vue.ts

```diff
@@ -10,7 +10,10 @@ import Slider from './shared/Slider.vue';
 import electron from 'electron';
 import { getPlatformService } from 'services/platforms';
 import { YoutubeService } from 'services/platforms/youtube';
+import VTooltip from 'v-tooltip';
 
+Vue.use(VTooltip);
+VTooltip.options.defaultContainer = '#mainWrapper';
```

Comment:
I think maybe we should only do these 2 lines once, instead of in every component that uses them.  Maybe we should add it in `app/app.ts`.